avalanche/training/supervised/er_ace.py

```diff
@@ -309,7 +157,8 @@ def _before_training_exp(self, **kwargs):
         # Update buffer before training exp so that we have current data in
         self.storage_policy.update(self, **kwargs)
         buffer = self.storage_policy.buffer
-        if len(buffer) >= self.batch_size_mem:
+        if len(buffer) >= self.batch_size_mem and \
```

Comment:
I also fixed the bad behavior of ER-ACE in non-online setting, now I make sure that both the buffer is big enough and the training experience is more than 0